lib/engine/game/g_1850/game.rb

```diff
@@ -331,9 +331,40 @@ def revenue_for(route, stops)
         end
 
         def sell_shares_and_change_price(bundle, allow_president_change: true, swap: nil, movement: nil)
-          @sell_queue << [bundle, bundle.corporation.owner]
+          bundle_owner = bundle.owner
+          @sell_queue << [bundle, bundle.corporation.owner] unless bundle.corporation.president?(bundle_owner)
 
```

Comment:
add logic here so that if president sells shares, they aren't asked to price protect those shares.